mdp/hinet/layer_online.py

```diff
@@ -0,0 +1,149 @@
+"""
+Module for OnlineLayers.
```

Comment:
Yes, I too had this as a thought initially. In fact in my forked mdp copy, you can find among my earlier commits that I had followed this style of separate online folder. But after discussing at length here at the INI group meetings, we concluded that this was not such a great idea :)